### DISCUSSION TITLE
Short course instructors for M&M 2024 in Cleveland, OH

### DISCUSSION THREAD

**Discussion by jat255** [hyperspy/hyperspy#3200] [General]
A call was recently announced for short course proposals for next year's M&M 2024 in Cleveland, OH. As we were not able to have a course this year at M&M, I think it's especially important we try to do one next year, so I'll be submitting on behalf of HyperSpy.

To that end, please leave a comment here if you would be willing to attend M&M next year and participate in the short course on July 28, 2024. Thanks!

I plan to be there.

**Comment by hakonanes** [comment]
That's good. While I cannot commit 100% I can go, I'd really like to contribute to such a workshop.

**Comment by CSSFrancis** [comment]
I probably cannot commit to anything as well.  If things go well I should graduate by then so it's a little up in the air.  That being said if I'm not going to be there I can probably have some of my group mates (@SyHuang19) help out. 

That being said I want to rewrite or adjust a fair bit of the workshop material. I also think that we should advertise it to both new and existing users because of a lot of the changes in the 2.0.0 release.

**Comment by magnunor** [comment]
Same here, I can't commit 100% at this time, but if I'm there I'll definitely join.

With regards to workshop material, I'll probably rework the EELS notebook since we got some new nice open source EELS edges now.

**Comment by CSSFrancis** [comment]
@jat255 Any more information on if this is happening?

**Reply by jat255** [reply]
Sorry @CSSFrancis, I just saw this... There will not be a short course this year at M&M. Hopefully next year, and/or Andy Herzing and I wanted to set up a more ind-depth course, perhaps as part of MAS's topical conference (TC) system.

**Reply by CSSFrancis** [reply]
@jat255 That sounds like a great idea! Let me know if you need any help with organizing something like that.
